app/konnect/service-catalog/index.md

```diff
@@ -4,17 +4,127 @@ subtitle: Track every service across your architecture
 content-type: explanation
 ---
 
-A new {{site.service_catalog_name}} experience is currently in development. The new {{site.service_catalog_name}} will be a one-stop place to see all your services, with all your service information, right at your fingertips. 
+{{site.konnect_saas}}'s Service Catalog offers a comprehensive catalog of all services running in your organization. This catalog gives you insight into your organization's services and their dependencies. It includes both {{site.base_gateway}} services and external integration resources.
 
-If you would like to get early access to the new {{site.service_catalog_name}}, go to **{{site.service_catalog_name}}** in the sidebar of your {{site.konnect_short_name}} instance.
+<!-- vale off-->
+{% mermaid %}
+graph LR
+  %% Define styles for icons and nodes
+  classDef konnect stroke-dasharray:3,rx:10,ry:10
+  classDef serviceCatalog fill:#d0e5f3,rx:10,ry:10
+  %% Define the resources
+  ExternalIntegration1(External Integration 1):::resource
+  ExternalIntegration2(External Integration 2</div>):::resource
+  GatewayManager(<div style="text-align:center;"><img src="/assets/images/logos/konglogo-gradient-secondary.svg" style="max-width:25px; display:block; margin:0 auto;" class="no-image-expand"/> Gateway Manager</div>):::resource
 
-## How to manage your services
+  %% Define the Konnect Service Catalog and its subgraphs
+  subgraph KonnectServiceCatalog[Konnect Service Catalog]
+    direction TB
 
-While the new {{site.service_catalog_name}} is in development, you can manage your services in the following ways:
+    %% Define the Billing Service subgraph inside Konnect Service Catalog
+    subgraph BillingService[Billing Service]
+      direction LR
+      KongGatewayService(Gateway service 1)
+      ExternalResourceMapping1(External resource)
+      ExternalResourceMapping2(External resource)
+    end
+
+    %% Define the Inventory Service Catalog Service subgraph inside Konnect Service Catalog
+    subgraph InventoryService[Inventory Service]
+      direction LR
+      SomeOtherService(Gateway service 2)
+      AnotherService(External resource)
+      DifferentRepo(External resource)
+    end
+
+  end
+
+  %% Connect the resources to the Konnect Service Catalog
+  ExternalIntegration1 -- "External resource" --> KonnectServiceCatalog
+  ExternalIntegration2 -- "External resource" --> KonnectServiceCatalog
+  GatewayManager -- "Gateway services" --> KonnectServiceCatalog
+
+  %% Style the subgraphs
+  class KonnectServiceCatalog konnect
+  class BillingService serviceCatalog
+  class InventoryService serviceCatalog
+{% endmermaid %}
+<!-- vale on-->
+
+> *Figure 1: This diagram shows how you can use both external integrations, like GitHub and PagerDuty, as well as built-in integrations like Gateway Manager to pull resources into Service Catalog. You can then map those resources (like GitHub repositories, PagerDuty services, and Gateway services) to Service Catalog services.*
```

Comment:
"...both external integrations, like GitHub and PagerDuty, as well as **_Konnect-internal_** integrations like Gateway Manager..."

I feel like using the word "internal" here better contrasts the use of the word "external" earlier in the sentence. Wha do you think?